### PR TITLE
Add explanation for Windows incompatibility in installation docs

### PR DESCRIPTION
**Pull request by ahill187**
## Description
Mention operating system compatibility in docs. As mentioned in this review: https://github.com/openjournals/joss-reviews/issues/2868#issuecomment-752687437, there are difficulties in installing on `Windows` machines. To address this, I have added a few lines in the installation docs to state that we don't currently support `Windows`, but would like to in the future.

## Linked issues
Related to: https://github.com/ivadomed/ivadomed/issues/604

### CODE REVIEW

**Review by charleygros — APPROVED** [review]
Sounds good to me! Thanks @ahill187

File: docs/source/installation.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Installation
 ============
 
+Supported OS:
+-------------
```
Comment (jcohenadad) [inline docs/source/installation.rst:5]: for consistency with other headings
```suggestion
Supported OS
------------
```

File: docs/source/installation.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+Currently, we only support ``MacOS`` and ``Linux`` operating systems. You can try installing
+on ``Windows``, but we cannot guarantee success. We would like to add support for ``Windows``
+in the future though.
```
Comment (jcohenadad) [inline docs/source/installation.rst:9]: More positive note:
```suggestion
Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows`` users have the possibility to install and use ``ivadomed`` using `Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_.
```

**Comment by jcohenadad** [comment]
@ahill187 i was in the middle of writing comments-- could you please consider them in a new PR (or revert the PR to add my comments)-- thx

**Comment by ahill187** [comment]
@jcohenadad ah sorry about that! yah I'll revert this PR and integrate your comments :)

**Pull request merged**